cosmic_ray/timing.py

```diff
@@ -39,18 +37,3 @@ def __enter__(self):
 
     def __exit__(self, ex_type, ex_value, ex_traceback):
         pass
-
-
-def run_baseline(test_runner, module_name, test_args):
```

Comment:
I'm a little bit concerned since the baseline is now timing one thing - the time to run the tests - while the worker is timing something else - the time to start a python process and run the tests. This is probably not a practical issue for most cases, but it's worth keeping in mind.